ui/apps/platform/src/Containers/ComplianceEnhanced/Status/ComplianceStatusPage.tsx

```diff
@@ -1,18 +1,43 @@
 import React from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { Redirect, Route, Switch } from 'react-router-dom';
+
+import {
+    complianceEnhancedStatusClustersPath,
+    complianceEnhancedStatusPath,
+    complianceEnhancedStatusProfilesPath,
+    complianceEnhancedStatusScansPath,
+} from 'routePaths';
 
-import { complianceEnhancedStatusPath } from 'routePaths';
 import ComplianceDashboardPage from './Dashboard/ComplianceDashboardPage';
+import ComplianceReportsClusterPage from './Reports/ComplianceReportsClusterPage';
+import ComplianceReportsProfilePage from './Reports/ComplianceReportsProfilePage';
+import ComplianceReportsScanPage from './Reports/ComplianceReportsScanPage';
 
 function ComplianceStatusPage() {
     return (
         <>
             <Switch>
+                <Route
```

Comment:
Nit: it isn't clear if the purpose of each of these routes is entirely to pass a `type` prop to `ComplianceReports`, but if it is, I think you could do away with the three additional page components with something like:

```tsx
<Route path={complianceEnhancedStatusClustersPath}>
    <ComplianceReports type='cluster' />
</Route>
```

Reply:
I was hesitant to create 3 separate component but all 3 should produce varied requests and return slightly different responses which is why I thought of separating them. But I'm open to ideas for managing this in a single component. I picture using the 3 parent components to fetch data, standardize its format, and then forward it to the `ComplianceReports` component.

Reply:
Good call - I wasn't sure if the queries would be similar enough or not, but if there are going to be three separate queries as well for each component, keeping them apart and doing the top level fetching is a good idea. 👍